woocommerce/woocommerce-gateway-stripe
# Remove Link Autofill

Fixes #3612

## Changes proposed in this Pull Request:

Removes Link Autofill Modal, as part of the move to Link in the Payment Element.







## Testing instructions
Objective: Test that Link autofill is no longer present, and nothing broke from the checkout process.

1. Enable Link as a payment method.
2. As a logged in shopper, add a product to cart and go to block checkout.
4. For logged in shoppers, we expect the email address field to be pre-filled. Verify that the Link button is no longer there.
5. Verify that you can finish purchasing your order without any problem.
6. Repeat steps 2-5 for shortcode checkout.

## Notes
- With us no longer passing the Link autofill beta flag, if Link is enabled as a payment method, customers should see Link in their Payment Element. This is expected. Improvements to end-to-end experience will be addressed in a separate work.

| User is signed out | User is signed out (expanded) | Prompt to sign up|
|-----------|-----------|-------------|
| <img width="300" alt="Screenshot 2024-11-19 at 10 01 34 AM" src="https://github.com/user-attachments/assets/630101fd-d02e-4899-8d7f-aa106509f427"> | <img width="611" alt="Screenshot 2024-11-19 at 10 02 17 AM" src="https://github.com/user-attachments/assets/e9275001-04b6-404c-bb43-bb0364a1f8a7">| <img width="611" alt="Screenshot 2024-11-19 at 10 15 56 AM" src="https://github.com/user-attachments/assets/ff066cff-d513-4bc7-960d-f8cdd59f19df">|







---


**Post merge**
